plutus-core/executables/Common.hs

```diff
@@ -280,7 +279,7 @@ parseInput ::
   -- | The output is either a UPLC or PLC program with annotation
   IO (p PLC.SourcePos)
 parseInput inp = do
-    bsContents <- BSL.fromStrict . encodeUtf8 . T.pack <$> getInput inp
+    bsContents <- T.pack <$> getInput inp
```

Comment:
Actually I think it would make sense to push this even further and make `getInput` return `Text`. To do that you can just use the corresponding functions from the `Data.Text.IO` module.